modules/text_server_adv/text_server_adv.cpp

```diff
@@ -5684,6 +5681,8 @@ bool TextServerAdvanced::_shaped_text_update_breaks(const RID &p_shaped) {
 					}
 				}
 			} else {
+				ubrk_setText(bi, data + _convert_pos_inv(sd, r_start), _convert_pos_inv(sd, r_end - r_start), &err);
```

Comment:
Not sure in what conditions `ubrk_setText` can fail, but probable better to keep it where it was in it's own `if (U_SUCCESS(err)) { ... }` block.

Reply:
Huh, somehow I missed it even _can_ fail. Agreed, should definitely revert to its own block 👍